content/en/docs/concepts/configuration/liveness-readiness-startup-probes.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Liveness, Readiness, and Startup Probes
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+There are currently 3 types of probes in Kubernetes:
+
+- [Liveness Probe](#liveness-probe)
+- [Readiness Probe](#readiness-probe)
+- [Startup Probe](#startup-probe)
+
+<!-- body -->
+
+## Liveness Probe
+
+Liveness probes determine when to restart a container. For example, liveness probes could catch a deadlock, where an application is running, but unable to make progress.
+
+If a Pod fails health-checks continuously, the Kubernetes terminates the Pod and starts a new one.
+
+Liveness probes do not wait for readiness probes to succeed. If you want to wait before executing a liveness probe you should use initialDelaySeconds or a startupProbe.
+
+
+## Readiness Probe
+
+Readiness probes determine when a container is ready to start accepting traffic. This is useful when waiting for an application to perform time-consuming initial tasks, such as establishing network connections, loading files, and warming caches. 
+
+If the readiness probe returns a failed state, the Kubernetes will remove the pod from all matching service endpoints.
```

Comment:
If the readiness probe returns a failed state, Kubernetes removes the pod from all matching service endpoints.

Reply:
Done.